next-app/src/routes/projects/[project_code]/activities/+server.ts

```diff
@@ -66,12 +71,13 @@ export async function fetch_activities({ cookie, start_date, end_date }: Activit
 	return activity.map(transform)
 }
 
-function transform({ id, action, date, content }: LegacyActivity): Activity {
+function transform({ id, action, date, content, userRef }: LegacyActivity): Activity {
 	return {
 		id,
 		action,
 		date,
-		user: content.user,
+		user: userRef.username,
+		avatar: userRef.avatar_ref,
```

Comment:
I'm wondering if we should make `user` an object with the `avatar` under it?

Reply:
I would be totally fine with that - probably makes more sense.  I added avatar because I would like it to show in the activity feed in a future re-working.

Reply:
That might be more intuitive. I'm guessing there would be a username field under it? That way the 'user' refers to something more than just the username (in the current case where it's just the username, maybe we should call it 'username' instead). Thoughts @megahirt ?

Reply:
👍 just amended the commit to this branch with this change